public/js/index.js

```diff
@@ -362,6 +363,17 @@ async function removeLesson(deleteId) {
 	}
 }
 
+modalLessonCloseByESC.addEventListener('keydown', event => {
+    if (event.key === 'Escape' || event.keyCode === 27) {
+        modalLessonTitle.innerHTML = '';
+		modalLessonContent.innerHTML = '';
+
+		modalLesson.setAttribute('hidden', '');
+		overlay.setAttribute('hidden', '');
+		overlay.classList.remove('dark');
```

Comment:
I would also suggest to call the function `handleCloseLessonModal()` here rather than copying its contents. This way if we ever add any changes to how the modal is closed, it only needs to be done in one place